app/_src/gateway/reference/configuration/configuration-3.6.x.md

```diff
@@ -972,6 +972,8 @@ Defines the TLS ciphers served by Nginx.
 
 Accepted values are `modern`, `intermediate`, `old`, `fips` or `custom`.
 
+If you want to enable TLSv1.1, this value has to be `old`.
```

Comment:
```suggestion
If you want to enable TLSv1.1, this value must be `old`.
```

Reply:
I have to ignore these unfortunately, since this is a generated file :(